birdhouse/default.env

```diff
@@ -83,24 +92,24 @@ export SERVER_LICENSE_URL='${__DEFAULT__SERVER_LICENSE_URL}'
 # Defaults for required variables recommended for override for security reasons.
 # Those will not be set explicitly as defaults to ensure they are overridden explicitly by the instance.
 # These values would be detected only if the instance was configured using a copy of 'env.local.example'.
-# Any default value change here should be synchronized with the corresponding value in 'env.local.example'.
-export __DEFAULT__MAGPIE_SECRET="itzaseekrit"
-export __DEFAULT__MAGPIE_ADMIN_USERNAME="admin"
-export __DEFAULT__MAGPIE_ADMIN_PASSWORD="qwertyqwerty!"
-export __DEFAULT__POSTGRES_PAVICS_USERNAME="postgres-pavics"
-export __DEFAULT__POSTGRES_PAVICS_PASSWORD="postgres-qwerty"
-export __DEFAULT__POSTGRES_MAGPIE_USERNAME="postgres-magpie"
-export __DEFAULT__POSTGRES_MAGPIE_PASSWORD="postgres-qwerty"
-export __DEFAULT__GEOSERVER_ADMIN_USER="admingeo"
-export __DEFAULT__GEOSERVER_ADMIN_PASSWORD="geoserverpass"
+# Any default value should use the corresponding '${__DEFAULT__{var}}' reference in 'env.local.example'.
```

Comment:
Will adjust the comment in the example to indicate where `__DEFAULT__` vars are defined.
This one however indicates that `__DEFAULT__` vars should be used (rather than literal values).
I can make it more explicit to say "to avoid literal duplication of default values".

Reply:
Done. 
See [`ebf070f` (#415)](https://github.com/bird-house/birdhouse-deploy/pull/415/commits/ebf070f5cb42be285ea24e3fd5cb3988ba4475fb)